WalletWasabi.Packager/Program.cs

```diff
@@ -584,7 +585,7 @@ private static void Publish()
 						"sudo mount -t drvfs C: /mnt/c -o metadata",
 						$"cd {linuxPath}",
 						$"sudo find {Tools.LinuxPath(newFolderRelativePath)} -type f -exec chmod 644 {{}} \\;",
-						$"sudo find {Tools.LinuxPath(newFolderRelativePath)} -type f \\( -name 'wassabee' -o -name 'hwi' -o -name 'bitcoind' \\) -exec chmod +x {{}} \\;",
+						$"sudo find {Tools.LinuxPath(newFolderRelativePath)} -type f \\( -name 'wassabee' -o -name 'hwi' -o -name 'bitcoind' -o -name 'tor' \\) -exec chmod +x {{}} \\;",
```

Comment:
If we change the permissions for Tor here (.deb package) then we should also do it in the creation of .tar.gz.

Side note: IMO `chmod +x` is wrong here because it gives permission for execution to everybody. That means that other users (or processes launched by other users) can have access to hwi interface or bitcoind process.  I would prefer `chmod u+x` instead.

Reply:
Very good point.